### PR TITLE
chore: extend nightly cross-browser test set (FPLA-2891)

### PR DESCRIPTION
**Pull request by Jamesnelson117**
**Before creating a pull request make sure that:**

- [x] PR's title includes JIRA ticket number and follow [Conventional Commits](https://www.conventionalcommits.org/) specification
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-2891

### Change description ###
Adds additional e2e to cross browser step of the nightly build pipeline

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -67,9 +67,9 @@ withNightlyPipeline(type, product, component) {
 
   loadVaultSecrets(integrationTestSecrets)
 
-  enableSecurityScan()
-  enableMutationTest()
-  enableFullFunctionalTest(120)
+//  enableSecurityScan()
+//  enableMutationTest()
+//  enableFullFunctionalTest(120)
```
Comment (Jamesnelson117) [inline Jenkinsfile_nightly:72]: Temporarily disabled to speed up testing in pipeline.

File: e2e/actors/base.js [hunk 2]
```diff
@@ -68,7 +68,6 @@ module.exports = {
         this.fillField('//input[@type="password"]', config.hmctsUser.password);
         this.wait(0.2);
         this.click('Sign in');
-        this.click('Yes');
```
Comment (tomaszpowroznik) [inline e2e/actors/base.js:71]: does not look right

File: e2e/actors/base.js [hunk 3]
```diff
@@ -272,7 +271,9 @@ module.exports = {
   async submitNewCaseWithData(data = mandatorySubmissionFields) {
     const caseId = await this.submitNewCase(config.swanseaLocalAuthorityUserOne);
     await apiHelper.populateWithData(caseId, data);
-    await this.refreshPage();
+    if (!this.isPuppeteer()) {
```
Comment (tomaszpowroznik) [inline e2e/actors/base.js:274]: is it not other way round?

File: package.json [hunk 4]
```diff
@@ -26,7 +26,7 @@
     "test": "codeceptjs run-multiple parallel",
     "test:functional": "MOCHAWESOME_REPORTFILENAME=functional codeceptjs run-multiple parallel --grep \"${TESTS_SELECTOR:=(?=.*)^(?!.*@smoke-tests)}\" --reporter mocha-multi",
     "test:smoke": "MOCHAWESOME_REPORTFILENAME=smoke codeceptjs run --grep \"${SMOKE_TESTS_SELECTOR:=@smoke-tests}\" --reporter mocha-multi",
-    "test:crossbrowser-e2e": "MOCHAWESOME_REPORTFILENAME=crossbrowser codeceptjs run-multiple --grep \"${SMOKE_TESTS_SELECTOR:=@smoke-tests}\" ${BROWSER_GROUP:-'--all'} -c saucelabs.conf.js --reporter mocha-multi",
+    "test:crossbrowser-e2e": "MOCHAWESOME_REPORTFILENAME=crossbrowser codeceptjs run-multiple --grep \"${SMOKE_TESTS_SELECTOR:=@cross-browser}\" ${BROWSER_GROUP:-'--all'} -c saucelabs.conf.js --reporter mocha-multi",
```
Comment (tomaszpowroznik) [inline package.json:29]: CROSS_BROWSER_TEST_SELECTOR instead of SMOKE_TESTS_SELECTOR or just 
--grep \"@cross-browser\"

**Review by tomaszpowroznik — APPROVED** [review]

**Review by Rob-Doherty — APPROVED** [review]
looks good to me, we can re-review the Safari and Edge upload testing in future if Saucelabs makes a change their end

**Pull request merged**